odoo-shippable/scripts/build-image.sh

```diff
@@ -161,7 +172,14 @@ EOF
 
 cat >> ~/.vimrc.before << EOF
 let g:spf13_bundle_groups = ['general', 'writing', 'odoovim',
-                           \ 'programming', 'php', 'ruby',
+                           \ 'programming', 'youcompleteme', 'php', 'ruby',
+                           \ 'python', 'javascript', 'html',
+                           \ 'misc']
+EOF
+
+cat >> ~/.vimrc.before.local << EOF
```

Comment:
Wait... the focus is avoid add 2 times the same lines of code... but I saw that `~/.vimrc.before` call to `~/.vimrc.before.local` then just append values to `~/.vimrc.before.local`

![screen shot 2017-01-20 at 9 22 30 am](https://cloud.githubusercontent.com/assets/6644187/22154456/40f4bfac-def1-11e6-9e05-05a73c0602c9.png)